README.md

```diff
@@ -22,7 +22,82 @@ Guidance on onboarding samples to docs.microsoft.com/samples: https://review.doc
 Taxonomies for products and languages: https://review.docs.microsoft.com/new-hope/information-architecture/metadata/taxonomies?branch=master
 -->
 
-Service Fabric can leverage the Managed Identity for Azure Active Directory (AD) to manage credentials in your application. These code examples show how to do so.
+This walkthrough steps through the process of deploying a service fabric cluster to Azure with managed identity enabled, and then deploying an application that uses managed identity to that cluster.
+
+## Prerequisites
+
+- [You will need Powershell and the Az Powershell library to run this sample.](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps)
+- You should have an Azure subscription and resource group you can write to. There is no way to test this sample locally.
+
+## Overview
+
+### Create a User-Assigned Managed Identity and Key Vault
+
+1. [Create a user assigned managed identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-manage-ua-identity-portal)
+    - Make sure you create it in the region you would like your cluster to be in

```

Comment:
Maybe it's me, but I prefer a less colloquial tone/form of addressing in official documentation. Prefer describing the limitation/restriction, rather than addressing the user directly in an imperative manner.

Reply:
I agree, I've reworded most of it to be object first (Key Vault should have) rather than user first (you should configur eKV)